pkg/registry/client_test.go

```diff
@@ -71,7 +71,8 @@ func (suite *RegistryClientTestSuite) SetupSuite() {
 
 	// Init test client
 	suite.RegistryClient = NewClient(&ClientOptions{
-		Out: suite.Out,
+		Debug: false,
```

Comment:
this can be removed as the default value for booleans is `false`.

Reply:
done